test/e2e/e2e.go

```diff
@@ -396,42 +457,10 @@ func LabelNamespace(clients *test.Clients, logf logging.FormatLogger, labels map
 	return err
 }
 
-// CreateNamespaceIfNeeded creates a new namespace if it does not exist
-func CreateNamespaceIfNeeded(t *testing.T, clients *test.Clients, logf logging.FormatLogger) (string, func()) {
-	shutdown := func() {}
-	ns := pkgTest.Flags.Namespace
-	logf("Namespace: %s", ns)
-
-	nsSpec, err := clients.Kube.Kube.CoreV1().Namespaces().Get(ns, metav1.GetOptions{})
-
-	if err != nil && errors.IsNotFound(err) {
-		nsSpec = &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: ns}}
-		logf("Creating Namespace: %s", ns)
-		nsSpec, err = clients.Kube.Kube.CoreV1().Namespaces().Create(nsSpec)
-
-		if err != nil {
-			t.Fatalf("Failed to create Namespace: %s; %v", ns, err)
-		} else {
-			shutdown = func() {
-				clients.Kube.Kube.CoreV1().Namespaces().Delete(nsSpec.Name, nil)
-				// TODO: this is a bit hacky but in order for the tests to work
```

Comment:
I'm guessing this no longer needed. Can you confirm?

Reply:
It's no longer needed.